echo_client.py

```diff
@@ -0,0 +1,28 @@
+import socket
+import sys
+
+
+def create_client_socket():
+    client_socket = socket.socket(
+        socket.AF_INET,
+        socket.SOCK_STREAM,
+        socket.IPPROTO_IP)
+    return client_socket
+
+
+def comm(client_socket):
+    client_socket.connect(('127.0.0.1', 50000))
+    # Send argv from command line
+    client_socket.sendall(', '.join(sys.argv[1:]))
+    client_socket.shutdown(socket.SHUT_WR)
+
+
+def receive(client_socket):
+    print client_socket.recv(32)
```

Comment:
you should never write a method that simply prints a value.  Always return the value and then you can print it from the calling context if you want to.  You can't *do* anything with printed values easily.